integration/awskms/src/aws_kms_client_test.rs

```diff
@@ -87,16 +86,15 @@ fn test_new_client_with_credentials_with_good_credentials_csv() {
 fn test_new_client_with_credentials_with_good_credentials_ini() {
     let uri_prefix =
         "aws-kms://arn:aws:kms:us-east-2:235739564943:key/3ee50705-5a82-4f5b-9753-05c4f473922f";
-    let path: PathBuf = [
+    let cred_ini_file: PathBuf = [
         env!("CARGO_MANIFEST_DIR"),
         "../../testdata",
         "credentials_aws.cred",
     ]
     .iter()
     .collect();
-    let cred_ini_file = path.to_str().unwrap();
     assert!(
-        AwsClient::new_with_credentials(uri_prefix, cred_ini_file).is_ok(),
+        AwsClient::new_with_credentials(uri_prefix, &cred_ini_file).is_ok(),
         "reject good CSV cred file"
```

Comment:
Pre-existing, but noticed now that this should probably say "INI" rather than "CSV".

Reply:
Fixed.